src/models/Type.ts

```diff
@@ -0,0 +1,15 @@
+import mongoose, { Schema, Document } from 'mongoose';
+
+import { IUser } from './User';
+
+export interface IType extends Document {
+    user: IUser['_id'];
+    name: string;
```

Comment:
Please add default: boolean

Reply:
I think `isDefault` is better, since default is a reserved keyword in js and ts. 
https://github.com/tsoibet/kaado/pull/2/commits/49e440052ad51ca33293cb8aa07ce22b9c95370f